src/components/ConfirmedActionButton/__tests__/ConfirmedActionButton.test.tsx

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { ConfirmedActionButton } from '../ConfirmedActionButton';
 
-const popupText = 'Are you sure?';
+const popupText = 'Are you sure you would like to delete the test sample?';
 
 describe('<ConfirmedActionButton />', () => {
   it('renders the popup when clicked', () => {
-    const { queryByText, getByText } = render(<ConfirmedActionButton size="large" type="accept" />);
+    const { queryByText, getByText } = render(
+      <ConfirmedActionButton
+        size="large"
+        type="accept"
+        displayText="Are you sure you would like to delete the test sample?"
```

Comment:
Is there a reason why this is copied instead of just using the value of `popupText`?

Reply:
Oops no reason, but not a big deal because we're gonna delete that component anyways.